### PR TITLE
Update design for Create / Load NFT on DAO dashboard

### PR DESCRIPTION
**Pull request by mudrila**
Extract `CreateNFTModalContent` to reusable form, and actually use it in modal and on the DAO dashboard

### CODE REVIEW

File: src/pages/DAO/index.tsx [hunk 1]
```diff
@@ -179,8 +183,11 @@ const DAOPage: FunctionComponent = () => {
 									push(`${pathname}?page=${nextPage}`)
 								}}
 							/>
-							{isAdmin && page === "nfts" && (
-								<DAOCollection gnosisAddress={dao.gnosisAddress} canEdit={isAdmin} />
+							{isAdmin && page === "createNFT" && (
+								<CreateNFTForm
+									gnosisAddress={dao.gnosisAddress}
+									domains={user?.myDomains as Domain[]}
```
Comment (Koochr) [inline src/pages/DAO/index.tsx:189]: instead of this insertion add a check that user is not nullish

**Review by Koochr — CHANGES_REQUESTED** [review]
The idea is correct, but:
1. Please remove "Create/Load NFT" modal from DAO collection page
2. Remove "success" stage from create NFT page and redirect user to "collection" instead
3. While doing p.3 make sure not to break success stage of create NFT modal (it is still used in user profile)
4. Also be sure that the new NFT appears on collection page after creation

File: src/components/DAO/DAOCollection/index.tsx [hunk 2]
```diff
@@ -1,34 +0,0 @@
...
-
-	const updateGallery = () => {
-		setGalleryKey(Math.random())
-	}
-
-	return (
-		<div className="dao-collection">
-			<div className="dao-collection__edit-buttons">
-				{canEdit && account && (
-					<CreateNFTModal
```
Comment (mudrila) [inline src/components/DAO/DAOCollection/index.tsx:22]: After removing `CreateNFTModal` from this page - the component became fairly too small to keep it. Thus I just extracted `NFTGallery` to DAO Dashboard

File: src/pages/DAO/index.tsx [hunk 3]
```diff
@@ -166,55 +169,57 @@ const DAOPage: FunctionComponent = () => {
 						)}
 					</div>
 					<div className="dao__main">
-						<>
```
Comment (mudrila) [inline src/pages/DAO/index.tsx:169]: We don't this `Fragment` here anymore

File: src/pages/DAO/index.tsx [hunk 4]
```diff
@@ -166,55 +169,57 @@ const DAOPage: FunctionComponent = () => {
...
+							currentPage={page}
+							onChange={nextPage => {
+								push(`${pathname}?page=${nextPage}`)
+							}}
+						/>
+						{isAdmin && page === "createNFT" && (
+							<CreateNFTForm
+								gnosisAddress={dao.gnosisAddress}
+								domains={user ? user.myDomains : []}
+								afterCreate={() => push(`${pathname}?page=collection`)}
```
Comment (mudrila) [inline src/pages/DAO/index.tsx:183]: `afterCreate` is called right before setting the stage to "success", so it does the trick to not show success message and rather immediately redirect to collection

**Pull request merged**